### DISCUSSION TITLE
Add some debug tools

### DISCUSSION THREAD

**Discussion by bruhcode68** [ruffle-rs/ruffle#11567] [Ideas]
there is already some information that can be shown like uncompressed length, swf version, if its actionscript 3 or not, but maybe some other cool debug tools, like viewing and changing values or maybe a save editor under save manager

maybe there is this already somewhere but it lowers the hassle if its just built in, if it exists already, it would be very helpful if you told me

**Comment by n0samu** [comment]
We have some arriving very soon! https://github.com/ruffle-rs/ruffle/pull/11419
